spec/lita/adapters/slack/rtm_connection_spec.rb

```diff
@@ -72,6 +78,16 @@ def with_websocket(subject, queue)
         expect(websocket).to be_an_instance_of(Faye::WebSocket::Client)
       end
     end
+
+    before do
+      config.proxy = proxy_url
+    end
```

Comment:
This `before` block and the test case below it need to be nested in another `context` block. As it is, the before block is applying to all the test cases within `describe "#run"`. I'd probably name the new context "with a proxy server specified". Then the test case can have the same name as the one above it: "creates the WebSocket".